### PR TITLE
Portainer

### PR DESCRIPTION
**Pull request by thedetukov**
Implement image "portainer"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# Launch
+
+```shell
+docker run --rm -it \
+--env PORTAINER_LIGHT_BG_COLOR='#ff0000' \
+--env PORTAINER_DARK_BG_COLOR='#00ff00' \
+--env PORTAINER_HIGH_BG_COLOR='#0000ff' \
+-p 8000:8000 -p 9000:9000 \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v portainer_data:/data ololo
```
Comment (theanurin) [inline README.md]: `ololo` ?

Comment (theanurin) [inline README.md]: No need `-v portainer_data:/data` for test launch

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* portainer v2.18.3-alpine
+
+# Launch
+
+```shell
+docker run --rm -it \
+--env PORTAINER_LIGHT_BG_COLOR='#ff0000' \
+--env PORTAINER_DARK_BG_COLOR='#00ff00' \
+--env PORTAINER_HIGH_BG_COLOR='#0000ff' \
+-p 8000:8000 -p 9000:9000 \
```
Comment (theanurin) [inline README.md]: Pls use `--publish` instead `-p` (for documentation we use long names)

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (thedetukov) [reply README.md]: Fixed, "ololo" changed on thedetukov/portainer.

Comment (thedetukov) [reply README.md]: Fixed this issue.

File: README.md (hunk 2 above)
Comment (thedetukov) [reply README.md]: Changed -p on --publish.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# Launch
+
+```shell
+docker run --rm -it \
+--env PORTAINER_LIGHT_BG_COLOR='#ff0000' \
+--env PORTAINER_DARK_BG_COLOR='#00ff00' \
+--env PORTAINER_HIGH_BG_COLOR='#0000ff' \
+--publish 8000:8000 \
+--publish 9000:9000 \
+-v /var/run/docker.sock:/var/run/docker.sock \
```
Comment (theanurin) [inline README.md]: Pls use long name version of `-v` (for documentation we use long names)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+# Inside
+
+* portainer v2.18.3-alpine
+
+# Launch
+
+```shell
+docker run --rm -it \
+--env PORTAINER_LIGHT_BG_COLOR='#ff0000' \
```
Comment (theanurin) [inline README.md]: Pls add two spaces before `--env`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+* `tcp/9000` - Portainer listening endpoint
+
+## Volumes
+
+* No any volumes
+
+# Inside
+
+* portainer v2.18.3-alpine
```
Comment (theanurin) [inline README.md]: Pls use capital first letter `portainer` -> `Portainer`

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (thedetukov) [reply README.md]: Changed -v on long name --version.

File: README.md (hunk 5 above)
Comment (thedetukov) [reply README.md]: Changed portainer -> Portainer.

File: README.md (hunk 4 above)
Comment (thedetukov) [reply README.md]: Added two spaces  before  --env and following lines.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+# Launch
+
+```shell
+docker run --rm -it \
```
Comment (theanurin) [inline README.md]: Pls use long name version of `-i` (for documentation we use long names)

Comment (theanurin) [inline README.md]: Pls use long name version of `-t` (for documentation we use long names)

**Review by theanurin — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (thedetukov) [reply README.md]: Changed -it -> --interactive --tty

Comment (thedetukov) [reply README.md]: Changed -it -> --interactive --tty .

File: docker/docker-entrypoint.sh [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+	echo
+
+	sed -i "s/--bg-body-color:var(--grey-9);/--bg-body-color:${PORTAINER_LIGHT_BG_COLOR};/g"     "${MAIN_CSS_FILE}"
+	sed -i "s/--bg-body-color:var(--grey-2);/--bg-body-color:${PORTAINER_DARK_BG_COLOR};/g"      "${MAIN_CSS_FILE}"
+	sed -i "s/--bg-body-color:var(--black-color);/--bg-body-color:${PORTAINER_HIGH_BG_COLOR};/g" "${MAIN_CSS_FILE}"
+
+	exec /portainer
+else
+	echo "This command cannot be executed because there are multiple specified files. Contact developers."
+exit 4
```
Comment (theanurin) [inline docker/docker-entrypoint.sh]: Pls add TAB symbol

**Review by theanurin — CHANGES_REQUESTED** [review]

File: docker/docker-entrypoint.sh (hunk 7 above)
Comment (thedetukov) [reply docker/docker-entrypoint.sh]: Fixed.

**Pull request merged**